airflow/providers/amazon/aws/operators/ecs.py

```diff
@@ -378,7 +387,10 @@ def _check_success_task(self) -> None:
             containers = task['containers']
             for container in containers:
                 if container.get('lastStatus') == 'STOPPED' and container['exitCode'] != 0:
-                    raise AirflowException(f'This task is not in success state {task}')
+                    last_logs = "\n".join(self._last_log_messages(self.number_logs_exception))
+                    raise AirflowException(
+                        f"This task is not in success state - last logs from Cloudwatch:\n{last_logs}"
+                    )
```

Comment:
How about including the count in the message?

```suggestion
                    raise AirflowException(
                        f"This task is not in success state - last {self.number_logs_exception} "
                        f"logs from Cloudwatch:\n{last_logs}"
                    )
```